auditlog/iter.go

```diff
@@ -0,0 +1,63 @@
+package auditlog
+
+type Iterable interface {
+	GetID() string
+}
+
+// Query is the function used to get a page listing.
+type Query func(ListRequestParams) ([]Iterable, ListMeta, error)
```

Comment:
I didn't catch this in previous reviews, but I'm wondering if `Query` is a little generic. What do you think about:
```suggestion
type IterableQuery func(ListRequestParams) ([]Iterable, ListMeta, error)
```